packages/main/src/themes/sap_fiori_3/parameters-bundle.css

```diff
@@ -53,5 +53,5 @@
 @import "./MultiComboBox-parameters.css";
 @import "./SliderBase-parameters.css";
 @import "../base/StepInput-parameters.css";
-@import "../base/sizes-parameters.css";
+@import "./sizes-parameters.css";
```

Comment:
maybe move the import so that it's not in between both imports from base/

Reply:
there's no such place, but I have moved it as the last line.